commander/src/bootstrapping/commands/config/create.ts

```diff
@@ -17,24 +17,22 @@ import { Command, flags as flagParser } from '@oclif/command';
 import fs from 'fs-extra';
 import { join, resolve } from 'path';
 import inquirer from 'inquirer';
-import { defaultConfig } from '../../utils/config';
+import { defaultConfig } from '../../../utils/config';
 
-export default class NetworkConfigCommand extends Command {
+export class CreateCommand extends Command {
 	static description = 'Creates network configuration file.';
 	static examples = [
-		'generate:network-config mydir',
-		'generate:network-config mydir --label alpha-sdk-app',
-		'generate:network-config mydir --label alpha-sdk-app --community-identifier sdk',
-	];
-	static args = [
-		{
-			name: 'networkName',
-			description: 'Directory where the config file is saved.',
-			required: true,
-		},
+		'generate:network-config --network-name mydir',
+		'generate:network-config --network-name mydir --label alpha-sdk-app',
+		'generate:network-config --network-name mydir --label alpha-sdk-app --community-identifier sdk',
 	];
 
 	static flags = {
+		'network-name': flagParser.string({
+			char: 'n',
+			description: 'Name of the network directory where the config file is saved',
+			default: '',
+		}),
 		label: flagParser.string({
 			char: 'l',
 			description: 'App Label',
```

Comment:
When we bootstrap app the app-lable default value is the directory name. Later that also been added to `package.json`. Is there a way to check if `cwd` contains a `package.json` then use its package name as default value.